libs/extensions/include/NanoMuon.hpp

```diff
@@ -28,6 +28,11 @@ class NanoMuon {
   inline float GetPt() { return physicsObject->Get("pt"); }
   inline float GetEta() { return physicsObject->Get("eta"); }
   inline float GetPhi() { return physicsObject->Get("phi"); }
+  inline float GetOuterEta() { return physicsObject->Get("outerEta"); }
+  inline float GetOuterPhi() { return physicsObject->Get("outerPhi"); }
+
+  float GetMatchIdxForNBestMatch(int N);
+  float GetMatchesForNBestMatch(int N);
```

Comment:
Just a guess, but is this actually supposed to mean more like "GetMatchIdxForNthBestMatch"?